### DISCUSSION TITLE
[Question]: Does this project support users from unsupported regions

### DISCUSSION THREAD

**Discussion by ranqiangjun** [danny-avila/LibreChat#3184] [Q&A]
### What is your question?

Does this project support users from unsupported regions if the services are deployed in a supported region? Any suggestions, or will the team consider this? Or does anyone here have to deal with the same issue?

### More Details

Background: a few employees from unsupported regions are using LibreChat with OpenAI. Per the screenshot below, they may be unable to use LibreChat soon.

See https://help.openai.com/en/articles/5347006-openai-api-supported-countries-and-territories

### What is the main subject of your question?

Other

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/5101533/c7c6b734-2d70-4739-a3c5-c36d636ab38d)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This mainly depends on where your server is hosted.

**Reply by ranqiangjun** [reply]
@danny-avila Thanks for your quick reply. We have an instance of LibreChat hosted in Singapore, which is on the supported regions/countries list, but we still got the email above.

**Reply by danny-avila** [reply]
that's really odd. there shouldn't be anything to trace back to any other region. It looks like a lot of people got this email en masse: https://www.reddit.com/r/OpenAI/comments/1dnic3p/our_data_shows_that_your_organization_has_api/

To answer your first question, LibreChat is free to use wherever you host it. If the host server is in Signapore, I would not worry about your API access being blocked due to LibreChat. just make sure all of your API keys are not being used in a non-supported regions

**Reply by danny-avila** [reply]
and from the writing, it looks like they will just "block" API access from the region, and not "ban" your account.
